src/hub/ShareClassManager.sol

```diff
@@ -250,14 +250,16 @@ contract ShareClassManager is Auth, IShareClassManager {
         EpochInvestAmounts storage epochAmounts = epochInvestAmounts[scId_][depositAssetId][nowIssueEpochId];
         epochAmounts.navPoolPerShare = navPoolPerShare;
 
-        issuedShareAmount = PricingLib.assetToShareAmount(
-            epochAmounts.approvedAssetAmount,
-            hubRegistry.decimals(depositAssetId),
-            hubRegistry.decimals(poolId),
-            epochAmounts.pricePoolPerAsset,
-            navPoolPerShare,
-            MathLib.Rounding.Down
-        ).toUint128();
+        issuedShareAmount = (navPoolPerShare.raw() != 0)
```

Comment:
Wouldn't it be cleaner to add `isZero()` to `D18`?

Reply:
Maybe even `isNotZero()`

Reply:
Yeah, we can add both then

Reply:
[1696187](https://github.com/centrifuge/protocol-v3/pull/440/commits/1696187c119aab664020f71d151d0217169b1907)